### ISSUE TITLE
Elementor Pro 2.5.0 (Beta 3 Release)

### ISSUE THREAD

**Issue by bainternet** [elementor/elementor#7526]
===========
## [Beta 3 UPDATED - PLEASE READ BELOW!](https://github.com/elementor/elementor/issues/7526#issuecomment-475566116)
===========

Hi Elementor testers,

Brace yourself - Elementor is going to re-define web design all over again!

Help us test Elementor Pro v2.5.0 Beta, which brings Elementor one step further in terms of design capabilities:

# Meet Motion Effects: 
A powerful set of controls for adding beautiful transitions and animations to your design.

Until now, web designers who wanted to incorporate animations and motion effects needed to use either external libraries and or custom code.

Elementor Pro 2.5 offers an all-in-one solution for creating parallax, animations and other motion effects - all seamlessly integrated into the Elementor editor.

Let’s go over the new exciting effects, so you can test it out by yourself and send us your feedback.

Before you test, please make sure you are using:

- Elementor Core v2.5.8
- Elementor Pro v2.5.0 beta

## Scrolling Effects
Use Scrolling Effects to create amazing animations and interactions when the user scrolls through the page. Here’s a list of the scrolling effects:

### Vertical Scroll

![vertical](https://user-images.githubusercontent.com/660572/54546611-b6b49000-49ac-11e9-942b-f7fa9c00d6f4.gif)

This is the classic parallax effect you’ve been waiting for. Vertical scroll makes the element move at a different speed than the page while scrolling, in the direction and speed of your choosing.

**Test flow:**

- Before testing, please insert one of Elementor’s pre-made templates.

- Click on any widget > Advanced tab.

- Go to Motion Effects > Switch on ‘Scrolling Effects’, and click on the pencil icon next to the ‘Vertical Scroll’ option.

- Under ‘Direction’, set it to up or down and notice the difference when you scroll.

- Play with the Speed scale and see the motion speed difference.

- Set the Viewport value between 50-100, and notice how the movement starts only after you scroll half the viewport.

> **Please note!** The viewport value is calculated according to the height of the viewport (meaning the visible size of the screen), and not to the height of the full page. This is true in all situations, except for elements with a fixed position. [See explanation image here](https://i.imgur.com/wCnWzAf.jpg)

### Horizontal Scroll

![Horizontal Scroll](https://user-images.githubusercontent.com/660572/54546620-b7e5bd00-49ac-11e9-9dcb-e10e28fcb2b3.gif)

Horizontal scroll means when the visitor scrolls up and down, the element moves right and left accordingly. An example of use - a section with clouds that moves to the right when the visitor scrolls down.

**Test flow:**

- Click on any section with a background image.
- Go to Style > Background > Switch on ‘Scrolling effects’, and click on the pencil icon next to the ‘Horizontal scroll’ option.
- Under ‘Direction’, set Left or Right and notice the difference while you scroll
- Play with the Speed scale and see the motion speed difference
- Set the Viewport between 0-50, and notice the effect stops when you reach 50% of the page.

### Transparency

![opacity2](https://user-images.githubusercontent.com/660572/54546617-b7e5bd00-49ac-11e9-964c-c12d4e87b6a2.gif)

Transparency lets you gradually make elements more transparent or more visible in correlation to the visitor’s scroll. An example is having a title appear and then dissolve in accordance with the scroll.

There are 4 possible effect directions:
Fade in - Meaning the element starts as transparent and gradually becomes visible.  
Fade out - The element starts as visible and gradually becomes transparent.
Fade out in - The element starts as visible, then fades out, then becomes visible again.
Fade in out - The element starts as transparent, then becomes visible, then transparent again.

**Test flow:**

- Click on any heading in your page.
- Go to Advanced > Motion effects, switch on ‘Scrolling Effects’, and click on the pencil icon next to the ‘Transparency’ option.
- Choose the direction and see how the heading fades in and out according to your choice. For example: choose Fade In-Out.
- Set your desired peak transparency level and see the effect.
- Set the viewport between 30-60, and notice what happens: the heading starts off transparent, reaches peak Transparency level at 30%, stays the same until 60%, and fades out until the end of the screen.

### Blur 

![blur-scale](https://user-images.githubusercontent.com/660572/54546609-b6b49000-49ac-11e9-8603-fbba5c99d3f2.gif)

This setting is similar to Transparency, only instead of transparency, the element gets blurred. A good example is background images, that get proper focus only when the user scrolls down.

**Test flow:**

- Click on any section in your page
- Go to Style > Motion effects, switch on ‘Scrolling Effects’, and click on the pencil icon next to the ‘Blur’ option.
- Choose the direction and see the heading fades in and out according to your choice. For example: choose Fade In
- Set your desired peak blur level and see the effect
- Set the Viewport between 0-50, and notice this happens: the heading starts off blurry, reaches a maximum focus at 50%.

### Rotate 

Here, the element rotates as you scroll. A nice example is seen in the rotation of stars in the image below:

![rotate](https://user-images.githubusercontent.com/660572/54546614-b74d2680-49ac-11e9-8998-02750c5d9200.gif)

**Test flow:**
- Add an icon widget and set it to Cog.
- Go to Advanced > Custom Positioning, and set the width to ‘Inline’.
- Go to Advanced > Motion Effects, switch on ‘Scrolling Effects’, and click on the pencil icon next to the ‘Rotate’ option.
- Choose the direction and check that the icon rotates in the proper direction.
- Set the speed of rotation.
- Set a viewport of 0-100, so we will see the movement throughout the scroll.

> **Please note!** The two settings, X & Y Anchor Points, determine the axis around which the element scales or rotates. If you set orientation left-top, the rotation will happen around the left-top point of the element. If you set orientation center-center, the rotation will rotate around its center, like a wheel. This setting is relevant only to the above ‘Rotate’ effect as well as the ‘Scale’ effect listed below. 

### Scale

![scale](https://user-images.githubusercontent.com/660572/54546612-b74d2680-49ac-11e9-9dd7-52403b937c69.gif)

Scale lets you grow and shrink elements according to scroll. Example: A background that grows while scrolling.

**Test flow:**

- Add a new section.
- Under Style, add an image to the background.
- Switch on ‘Scrolling Effects’, and click on the pencil icon next to the Scale option.
- Play with the Direction, Speed & Viewport and make sure the effect works.

> **Please note!** Use the ‘Apply effects on’ to determine if motion effects are applied on mobile, desktop or tablet. Mouse effects will only affect desktop devices.

### Mouse Track
Create a sense of depth by making elements move in relation to the visitor’s mouse movement.

![mouse](https://user-images.githubusercontent.com/660572/54546616-b74d2680-49ac-11e9-85c5-fbed96bbaf0a.gif)

**Test flow:**

- Place 3 widgets in the same section.
- For each widget, go to Advanced > Motion Effects, and switch on ‘Mouse Effects’. Then, click on the pencil icon next to the ‘Mouse Track’ option.
- Choose the direction you want the widget to correspond with: ‘Direct’ or ‘Opposite’ to the direction of the mouse movement.
- Set the speed between 0-10.
- If you are up to the challenge, try to create a depth effect by making several elements move in relation to the visitor’s mouse movement.

### 3D Tilt

Similarly to how Mouse Track moves the element in relation to the mouse movement, the 3D Tilt effect tilts the element according to the same movement of the cursor.

![3d](https://user-images.githubusercontent.com/660572/54546613-b74d2680-49ac-11e9-828e-8f1fc76cd7d1.gif)

**Test flow:**

- Place an image in a column.
- Go to Advanced > Motion Effects, and switch on ‘Mouse Effects’. Then, click on the pencil icon next to the 3D Tilt option.
- Choose the direction you want the widget to correspond with: ‘Direct’ or ‘Opposite’ to the direction of the mouse movement.
- Set the speed between 0-10.
- Notice how the image tilts in accordance with your mouse movement and direction.
- If you are up to the challenge, try to create a depth effect by making several elements move in relation to the visitor’s mouse movement.

### Motion Effects Browser Compatibility
| Chrome  | Firefox | Safari | Opera | Edge | Internet Explorer |
| :----: |:----:| :-----:|:---:|:---:|:---:|
| Supported | Supported | Supported | Supported | Supported * | Not Supported |

\* Slightly laggy behavior
 
## Revamped Query Control

The new and improved Query Control will let you create amazing and advanced websites!

For example, let’s say you created a real estate website, with custom taxonomies like city, number of rooms, property size and so on.

Now, you can specify elaborate queries when deciding how to display posts:
Display only properties in London, excluding 2 rooms and below, with a property size of 80m and above. This feature gives you full control over your Posts, Portfolios and Products widgets.

Until now, users who wanted to create a custom posts query grid ran into some difficulties, which required custom code. 

Elementor Pro 2.5.0 introduces the revamped Query Control, which is incorporated into the Posts, Portfolio and Products widgets, enabling you to include or exclude posts based on any queryable term, filter by custom date query, and now also supports showing or hiding sticky posts. 

Additionally, we added the support for any related query, based on the same advanced query options.

Using the Products widget, now you can filter WooCommerce products based on specific attribute values, meaning you can show only products with, e.g. a red color. 

**Test Flow:**

- Import dummy posts (with tags and categories) or create 12 posts with several categories and tags.
- Create a new page and place the Posts widget.
- Go to Content > Query tab, include a certain category, and exclude a different tag.
- Check to see that the Posts widget shows the properly filtered posts.

--------------

### More Improvements

Please don't miss these points:

- Note: This beta requires Elementor v2.5.8
- Tweak: Added option to open a Popup by a custom selector (#6871, #6876, #7258)
- Tweak: Changed "Scrolling Effects" to "Motion Effects"
- Tweak: Added full border radius control options inside Popup
- Tweak: Show only one popup in its own preview
- Tweak: Conditions - Singular All Pages string changed to Pages
- Tweak: Added form field shortcode support for Drip tags (#7000)
- Tweak: Popups - Changed exit intent icon
- Tweak: Added dynamic capabilities to Price List widget (#7258)
- Tweak: Use filter get_meta_viewport for header templates (#7043)
- Tweak: Added Dynamic capabilities to Custom Attributes (#6779)
- Tweak: Nav Menu - decrease z-index (#6869)
- Tweak: Added dynamic capabilities to Flip Box widget (#6986)
- Tweak: Added responsive support to Popup entrance animation control
- Tweak: Added responsive support to WC Products Columns & Rows Gap controls (#6913)
- Tweak: use filterable Util::get_public_post_types() in Theme Builder (#7172)
- Tweak: WC Menu cart "View Cart" & "Checkout" buttons styling
- Tweak: Added Cloudflare rocket-loader support

#### Fixes:

- Fix: Conflict between archive-products widget and WC customizer
- Fix: Custom ID reset to default when dragging repeater
- Fix: Add to Cart widget spacing and space-between
- Fix: Library view when creating a new Header or Footer

--------------

### How to Install the Beta Version

If you still haven't enabled beta testing in Elementor, please follow these steps:

- Go to Elementor > Tools > Versions
- Enable the beta tester feature
- Go to Dashboard > Updates and update to the latest beta version.

This will get you set up to test the new Elementor Pro v2.5.0 Beta.
 

### Notes

**For bugs and remarks, please reply with a comment on the specific issue here on GitHub.**

Please keep this related to the Current beta Only! Any other issue or feature request, feel free to open a new issue.

Designated time for release: `March 26, 2019 (Estimated)`

Many thanks for your support and help!

Cheers,
The Elementor Team

**Comment by Hansol69** [comment]
WOW-WOW-WOW- This is just crazy. Im so impressed right now!

**Comment by stickypages** [comment]
For the new Posts Query, can we have the Term be Dynamic input?

**Comment by medvedbne** [comment]
So how many more js files and added weight will be loaded with this version?

**Comment by jofranlirio** [comment]
Please, add shortcodes like WP Astra in your components.

**Comment by rony747** [comment]
When will we get custom query designer. ?

**Comment by tudormnt** [comment]
Getting an error right off the bat:

Fatal error: Uncaught Error: Call to undefined method Elementor\Utils::print_js_config() in /app/public/wp-content/plugins/elementor-pro/includes/admin.php on line 61

**Comment by JourneysCT** [comment]
I may be experiencing an issue. I have a background image for a section. Then, I have text placed over the background. However, by applying the blur Motion Effect to the section itself, it blurs the image AND the text. I'm getting similar effects with other options as well.

Also, on the scale motion effect, the Hero Section is below the header and menu on the page. But when I use the Scale effect, the image can cover the Menu & entire header. I've tried playing with the -index of the section, but I'm not seeing any change.

**Comment by JourneysCT** [comment]
I have a section at the top of the page, under the menu and logo with a background image a part of the section. I then have two headlines placed over the section background. When I apply the blur motion effect on the section, it also effects the 2 headlines. Is there a way to separate the Headlines from the Background image?

Also, with the scale motion effect applied. I can make the Hero Section cover the entire Header section. I've experimented with z-index but that doesn't change anything.

I like the changes I'm seeing. Thank You!

**Comment by sunnyt7** [comment]
Nice updated! Could you clarify the workflow of:

"Tweak: Added option to open a Popup by a custom selector"

I'm not seeing the ability to do this anywhere.

Thanks!

**Comment by shafeekhamza** [comment]
creative ides are overflowing ..

awesome.

**Comment by followeyes** [comment]
Wooooooooow... real nice.

**Comment by deckerweb** [comment]
I get 2 notices in Query Monitor with Pro 2.5.0 Beta 1, for example when I am in the template list table overview (edit.php?post_type=elementor_library&tabs_group=library):

1. Trying to get property 'labels' of non-object --> wp-content/plugins/elementor-pro/modules/theme-builder/module.php:215
2. Trying to get property 'singular_name' of non-object --> wp-content/plugins/elementor-pro/modules/theme-builder/module.php:215

**Comment by shilo-ey** [comment]
[**@stickypages**](https://github.com/stickypages) commented on [Mar 18, 2019, 8:40 PM GMT+2](https://github.com/elementor/elementor/issues/7526#issuecomment-474049268 "2019-03-18T18:40:49Z - Replied by Github Reply Comments"):
> For the new Posts Query, can we have the Term be Dynamic input?
> When creating dynamic pages it's nice to be able to select testimonials, or portfolio items by the current page but I would  have to do this manually per page, instead of having a "single" template for them all. Make sense?

You can choose "Related" under the "Source" field. That would achieve the same behavior (relative to the post/page you're in).


[**@medvedbne**](https://github.com/medvedbne) commented on [Mar 18, 2019, 9:19 PM GMT+2](https://github.com/elementor/elementor/issues/7526#issuecomment-474063553 "2019-03-18T19:19:56Z - Replied by Github Reply Comments"):
> So how many more js files and added weight will be loaded with this version?

We've written our own slim library for the Motion Effects, therefore it shouldn't affect your loading time (has minor weight and only if Effects are activated in the page).




[**@sunnyt7**](https://github.com/sunnyt7) commented on [Mar 19, 2019, 6:41 AM GMT+2](https://github.com/elementor/elementor/issues/7526#issuecomment-474196062 "2019-03-19T04:41:57Z - Replied by Github Reply Comments"):
> "Tweak: Added option to open a Popup by a custom selector"
> I'm not seeing the ability to do this anywhere.

Popup > Advanced tab
![image](https://user-images.githubusercontent.com/45009641/54588939-ffac2900-4a2c-11e9-9a22-b224da495736.png)



[**@deckerweb**](https://github.com/deckerweb) commented on [Mar 19, 2019, 9:42 AM GMT+2](https://github.com/elementor/elementor/issues/7526#issuecomment-474231357 "2019-03-19T07:42:38Z - Replied by Github Reply Comments"):
> I get 2 notices in Query Monitor with Pro 2.5.0 Beta 1, for example when I am in the template list table overview (edit.php?post_type=elementor_library&tabs_group=library):
> 
> 1. Trying to get property 'labels' of non-object --> wp-content/plugins/elementor-pro/modules/theme-builder/module.php:215
> 2. Trying to get property 'singular_name' of non-object --> wp-content/plugins/elementor-pro/modules/theme-builder/module.php:215

Thanks, will take a look
[**@JourneysCT**](https://github.com/JourneysCT) commented on [Mar 18, 2019, 11:36 PM GMT+2](https://github.com/elementor/elementor/issues/7526#issuecomment-474111342 "2019-03-18T21:36:56Z - Replied by Github Reply Comments"):
> I have a section at the top of the page, under the menu and logo with a background image a part of the section. I then have two headlines placed over the section background. When I apply the blur motion effect on the section, it also effects the 2 headlines. Is there a way to separate the Headlines from the Background image?
> 
> Also, with the scale motion effect applied. I can make the Hero Section cover the entire Header section. I've experimented with z-index but that doesn't change anything.

- Blur issue - Make sure the blur effect you are making is in the style tab of the section > Background and not under Advanced tab (the entire section)
- Scale motion - Can you please elaborate? how can I reproduce your issue?

Thanks again for everyone!

**Comment by KiQdev** [comment]
Shouldn´t the motion effects be per section? If I move my mouse anywhere on the page where the section is, it moves.

**Comment by Lorenzoup** [comment]
#965

**Comment by jonatansmaia** [comment]
I can not update by wordpress on localhost:
PCLZIP_ERR_BAD_FORMAT (-10): Unable to find End of Central Dir Record signature
Do you have a link to download?

**Comment by shilo-ey** [comment]
We just released a new beta version, please update your version to v2.5.0 Beta 2.

This beta release includes:
- Fix: Added Elementor version compatibility validation
- Tweak: Option to count when Popup is closed in "Show up to X times" Advanced Rule
- Fix: Post types labels missing on Add New Template modal


**Please keep this related to the Current beta Only! any other [Issue Report](https://github.com/pojome/elementor/issues/new?template=Bug_report.md) or [Feature Request](https://github.com/pojome/elementor/issues/new?template=Feature_request.md) feel free to open a new issue.**

Thanks again to everyone.

**Comment by sunnyt7** [comment]
@shilo-ey 

> Popup > Advanced tab

Thank you!

If I had to make a suggestion, it would be to move this to the Triggers tab. That was the only place I naturally looked for it. I think the 'trigger by selector' use-case will also be much more common than 'trigger by number of clicks'... which I actually can't imagine why anyone would use.

> We've written our own slim library for the Motion Effects, therefore it shouldn't affect your loading time (has minor weight and only if Effects are activated in the page).

Hmm, this is interesting. Someone mentioned that frontend.js got 20kb bigger since the last stable release, suggesting that the library just went in there and gets loaded whether effects are activated or not.

Is the library being loaded somewhere else?

**Comment by shilo-ey** [comment]
[**@sunnyt7**](https://github.com/sunnyt7) commented on [Mar 19, 2019, 4:58 PM GMT+2](https://github.com/elementor/elementor/issues/7526#issuecomment-474408929 "2019-03-19T14:58:52Z - Replied by Github Reply Comments"):
> If I had to make a suggestion, it would be to move this to the Triggers tab. That was the only place I naturally looked for it. I think the 'trigger by selector' use-case will also be much more common than 'trigger by number of clicks'... which I actually can't imagine why anyone would use.

Thanks, but under the Advanced tab, there is a set of rules that can affect the popup regardless of the triggers and advanced rules. therefore this is the right place for it.

> > We've written our own slim library for the Motion Effects, therefore it shouldn't affect your loading time (has minor weight and only if Effects are activated in the page).
> 
> Hmm, this is interesting. Someone mentioned that frontend.js got 20kb bigger since the last stable release, suggesting that the library just went in there and gets loaded whether effects are activated or not.
> 
> Is the library being loaded somewhere else?

Currently no, just when you want to use the motion effects. You can [check Adam Preisers](https://www.youtube.com/watch?v=0gRUtecXq7s) comparison to the page speed performance before and after.

It should be summed up to about 4KB additional code weight when applied.

**Comment by vincentjflorio** [comment]
Serious question (and if this isn't the case, please don't mistake this as a repeat feature request), but if we can include and exclude by term, does that mean things like categories can finally be displayed in the default skins themselves? 

I'm only asking because I'm not sure if I'm supposed to be seeing it and it isn't working for me, or if we're now just able to _filter_ by category through inclusion/exclusion, and not actually _identify_ that successful filtering explicitly by label on the individual posts/cards themselves.

(such a cool feature, by the way, the term search is working great here =)

**Comment by MartinSchenkOnline** [comment]
very nice feature

**Comment by Deanphillips** [comment]
I agree with @sunnyt7 "I think the 'trigger by selector' use-case will also be much more common than 'trigger by number of clicks'"

**Comment by kochhase** [comment]
Hi very nice feature. I am be able to to make a starting page like?
https://aiteko.wip-themes.com/

**Comment by shilo-ey** [comment]
[**@vincentjflorio**](https://github.com/vincentjflorio) commented on [Mar 19, 2019, 6:42 PM GMT+2](https://github.com/elementor/elementor/issues/7526#issuecomment-474462519 "2019-03-19T16:42:06Z - Replied by Github Reply Comments"):
> Serious question (and if this isn't the case, please don't mistake this as a repeat feature request), but if we can include and exclude by term, does that mean things like categories can finally be displayed in the default skins themselves? 
> 
> I'm only asking because I'm not sure if I'm supposed to be seeing it and it isn't working for me, or if we're now just able to _filter_ by category through inclusion/exclusion, and not actually _identify_ that successful filtering explicitly by label on the individual posts/cards themselves.
> 
> (such a cool feature, by the way, the term search is working great here =)

Hi @vincentjflorio , Thanks for your feedback.
Currently, you can retrieve posts **and not** categories/terms. 
You can filter posts by the terms you wish to filter by (related, exclude, include).


[**@kochhase**](https://github.com/kochhase) commented on [Mar 20, 2019, 1:30 PM GMT+2](https://github.com/elementor/elementor/issues/7526#issuecomment-474791409 "2019-03-20T11:30:15Z - Replied by Github Reply Comments"):
> Hi very nice feature. I am be able to to make a starting page like?
> https://aiteko.wip-themes.com/

Hi @kochhase, no - the Motion Effects are triggered by scroll/mouse move and not an independent animation. 

Thanks

**Comment by misslaidlaw** [comment]
It feels like it's missing scroll snapping for sections.

**Comment by stefanocavaliere** [comment]
Unfortunately Popups stopped working with the last beta...

**Comment by fracii** [comment]
Dynamic Color in this release?? #6923

**Comment by alriksson** [comment]
And dynamic fields for more elements and [#6237](https://github.com/elementor/elementor/issues/6237) and group/repeater fields support please [#5979](https://github.com/elementor/elementor/issues/5979)

**Comment by vincentjflorio** [comment]
Thanks for the quick response @shilo-ey ; I'd seen it in practice in Elementor Extras and I got overeager that it might finally be in core; almost every site we build people want to show it in their post meta. We appreciate all the work you do to chock this thing full of literally everything else though =)

**Comment by shilo-ey** [comment]
We just released a new beta version, please update your version to v2.5.0 Beta 3.

This beta release includes:
- Tweak: Show Motion Effects controls in the background only if an image or gradient has been selected
- Fix: Added missing `countTimes` method to popup document.js fixes beta2 (#7549)
- Fix: Background update
- Fix: Fixed Column selector in Motion Effects
- Fix: Transform Origin affecting 3D Tilt when Scrolling Effects turned off

**Please keep this related to the Current beta Only! any other [Issue Report](https://github.com/pojome/elementor/issues/new?template=Bug_report.md) or [Feature Request](https://github.com/pojome/elementor/issues/new?template=Feature_request.md) feel free to open a new issue.**

Thanks again to everyone.

**Comment by jus303** [comment]
Hi Ben and the gang. I trying out the motion effects in beta! Loving it! I'm not sure but is this correct? If you are trying to add a motion effect to a fixed section or an element inside a fixed section - then they do no work. For example I have a logo in a fixed header, on scroll I want it to fade away - this will not work as its in a fixed header section...

**Comment by silvioghedin** [comment]
Sorry, but I have to agree with @sunnyt7 , it makes more sense to place the custom selector to the Trigger tab. I lost couple time trying to figure it out where it was this.

**Comment by Webtica** [comment]
Hi,

When testing Elementor PRO 2.5.0 beta 3 when using post  archive pages on template and use current query it shows all posts instead of only the posts in that category. When reverting back to elementor pro 2.4.7 the issue is fixed. So this would be a beta bug. 

If any more info is needed please let me know.

**Comment by shilo-ey** [comment]
[**@Webtica**](https://github.com/Webtica) commented on [Mar 25, 2019, 1:59 PM GMT+2](https://github.com/elementor/elementor/issues/7526#issuecomment-476165943 "2019-03-25T11:59:15Z - Replied by Github Reply Comments"):
> Hi,
> 
> When testing Elementor PRO 2.5.0 beta 3 when using post  archive pages on template and use current query it shows all posts instead of only the posts in that category. When reverting back to elementor pro 2.4.7 the issue is fixed. So this would be a beta bug. 
> 
> If any more info is needed please let me know.

Thanks! this issue will be fixed soon

**Comment by bainternet** [comment]
Elementor Pro V2.5.0 is out.

thank you all for helping out.

**Comment by JulGoleva** [comment]
> Fix: Conflict between archive-products widget and WC customizer
Super! Only now the filters for the category archive have stopped working.

**Comment by JulGoleva** [comment]
I create an Archive of products for the product category in the theme builder. I want to apply filters to it. But filtered products show the entire range of the store, and only products from this category should be displayed.

**Comment by fredarend** [comment]
Please, see this!
 [#6923](https://github.com/elementor/elementor/issues/6923)
